File: Day-50/CheckRotatedSorted.js
var check = function (nums) {
  let count = 0;
  for (let i = 1; i < nums.length; i++) {
    if (nums[i - 1] > nums[i]) {
      count++;
    }
  }
  if (nums[nums.length - 1] > nums[0]) {
    count++;
  }

  return count === 1 || count === 0 ? true : false;
};

check([1, 1, 1]);

File: Day-50/CheckRotatedSorted.py
def Check(arr):
    count = 0 
    for i in range(1, len(arr)):
        if(arr[i-1]>arr[i]):
            count += 1
    
    if(arr[len(arr)-1]>arr[0]):
        count += 1
    
    if(count <= 1):
        return True
    return False

Check([1,1,1])